{{cookiecutter.__project_slug}}/README.md

```diff
@@ -0,0 +1,169 @@
+
+### Installation
+
+To install this software first make a local copy of this repostitory (e.g by cloning it with `git clone`). In the root directory of your local folder run the following commands:
+
+``` bash
+poetry install
+```
+
+### Using the script
+
+#### The input
+
+The script expects a csv file as input. The first column of this file should be the names of the columns.
+
+#### Settings
+
+The script has a few settings which can be changed. These settings can be found at the top of the file `{{ cookiecutter.module_name }}/main.py`.
+
+| Setting | Explanation | Default |
+| --- |  --- | --- |
+| INPUT_FILE_PATH | The script looks for the input file in this location. | './example_input.csv' |
+| OUTPUT_PATH | The script creates the output file in this location. | './output.csv' |
+| COLUMN_NAME | The name of the column on the filtering should happen. | 'column_1' |
+| THRESHOLD | Rows with a value higher than this threshold in the column with the name `COLUMN_NAME` will be removed. | 5 |
+
+#### Running the script
+
+By running the `poetry install` command poetry created a [virtual environment](https://docs.python.org/3/library/venv.html). To run the script you need to use this virtual environment. It can be activated by running:
+
+```bash
+poetry shell
+```
+
+Now all the commands you run will be running in an environment with the correct versions of packages installed. At the same time your normal python installation will remain unaffected. If you want to exit this enviroment you can use the `deactivate` command.
+
+To now run the script run the following command:
+
+```bash
+python -m {{ cookiecutter.module_name }}.main
+```
+
+### The output
+
+The output is a new csv file with the rows filtered out. g.The file can be found in the location defined by the `OUTPUT_PATH` setting.
+
+## Developer Documentation
+<!--
+Provides some information for people who want to continue developing this software (or for yourself in the future if you ever return to it). It should provide the commnands a developer needs to for example test the software. Extend this section with specific instructions for developers of your software.
+-->
+
+### Running the tests
+
+To run the tests run the following command after running `poetry shell`
+
+```bash
+pytest
+```
+
+### Checking the codestyle
+
+To check if new code adheres to our code style you can run flake8 with the following command:
+
+```bash
+flake8 .
+```
+
+### Managing dependencies
+
+To add a new package that is reqequired use:
+
+```bash
+poetry add <package_name>
+```
+
+To update the dependencies run:
+
+```bash
+poetry update
+```
+
+If you want other people to be able to run the script with the same updated packages (for exampel to reproduce your results) then make sure to commit the changes to your `poetry.lock` file.
```

Comment:
I can imagine people do not know exactly what poetry lock file is and how to commit the changes to it